### DISCUSSION TITLE
What about a Folder for Archived Repositories ?

### DISCUSSION THREAD

**Discussion by srilakshmikanthanp** [community/community#15870] [Other Feature Feedback, Questions, & Ideas] [answered]
Hai, What about a Folder for Archived Repositories like WhatsApp Archived Chats ? So it is easy to look up the archived and non archived Repositories ?

![Archive Folder](https://user-images.githubusercontent.com/56999577/166131598-138c9394-c639-4aec-9ad4-c75e7953aa6b.png)

**Comment by PranavGPR** [comment]
It will be great. When we have lots of repositories and want to maintain them, it will be very useful too.

**Reply by Lanstan** [reply]
Да,это так есть ,что без надобности не было б бы спрососа.

**Reply by krissh-codes** [reply]
> Да,это так есть ,что без надобности не было б бы спрососа.
> 

டேய் எல்லாருக்கும் புரியுற மாதிரி பேசுடா.

**Comment by krissh-codes** [comment, marked as answer]
![Screenshot_20220502-074318_Chrome.png](https://user-images.githubusercontent.com/41967554/166176529-7aec98b4-18e7-4821-a4cd-424bffb79579.png)

You can already apply a filter to achieve your desired result.
